app/src/main.ts

```diff
@@ -52,13 +71,13 @@ async function Startup() {
         ],
     }).catch((err) => {
         console.error(err);
-        Sentry.captureException(err);
+        reportError(err);
```

Comment:
If we want to not use the `reportError` function, this could be:
```suggestion
        Sentry?.captureException(err);
```